app/controllers/admin/users_controller.rb

```diff
@@ -1,5 +1,16 @@
 class Admin::UsersController < Admin::BaseController
     def index
-        @users = User.all 
+        @default_users = User.default
```

Comment:
Don't need to change now but if this returns multiple default users I would like the method to be pluralized somehow maybe `default_users`